### ISSUE TITLE
Start of the rains and filters

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4520]
The start of the rains dialogue currently seems to ignore filters.
Is this the case with all the climatic events dialogues?

**Comment by dannyparsons** [comment]
@shadrackkibet can you test? This shouldn't be the case as the calculation system uses the current filter.

**Comment by shadrackkibet** [comment]
Yes that is true. I have checked in other climatic dialogs that uses calculation system and all ignore the current filter.

**Comment by dannyparsons** [comment]
Ok thanks, I'll look into that.
